src/editor/godot_kotlin_jvm_editor.cpp

```diff
@@ -143,11 +143,20 @@ void GodotKotlinJvmEditor::_notificationv(int p_notification, bool p_reversed) {
 
         menu_pop_up->add_item("About Godot Kotlin JVM");
         editor_base_control->add_child(about_dialog);
+        editor_base_control->add_child(error_dialog);
 
         FileSystemDock* file_system_dock = get_editor_interface()->get_file_system_dock();
         file_system_dock->connect("files_moved", this, "on_file_system_dock_file_moved");
         file_system_dock->connect("file_removed", this, "on_file_system_dock_file_removed");
         file_system_dock->connect("folder_moved", this, "on_file_system_dock_folder_moved");
+
+        if (OS::get_singleton()->get_environment("JAVA_HOME").empty()) {
```

Comment:
We are technically making the same check here and at `GDKotlin` but displaying the error message differently. What do you think about doing the initialization checks in `GDKotlin` exclusively? Then we can just expose a `GDKotlin::is_initialized()` method that returns true if the module is properly initialized. For error messages, we can just expose `GDKotlin::get_init_errors()` or something then display whatever string it returns.

Reply:
Agreed. Will change that